lib/kernel/test/zlib_SUITE_data/zipdoc

```diff
@@ -767,7 +767,7 @@ General Format of a ZIP file
             Currently, date-time width is 32-bit, but future version may
             support be 64-bit times (see flags)
 
-          CrGMTOffs     Long(signed!)   difference "local Creat. time - UTC"
+          CrGMTOffs     Long(signed!)   difference "local Create. time - UTC"
```

Comment:
This change causes three test cases in `zlib_SUITE` to fail. Please undo the change.